lib/srv/db/postgres/proxy.go

```diff
@@ -40,8 +39,10 @@ type Proxy struct {
 	TLSConfig *tls.Config
 	// Middleware is the auth middleware.
 	Middleware *auth.Middleware
-	// ConnectToSite is used to connect to remote database server over reverse tunnel.
-	ConnectToSite func(context.Context) (net.Conn, error)
+	// ConnectToService is used to connect to remote database server over reverse tunnel.
+	ConnectToService func(context.Context) (net.Conn, error)
```

Comment:
Why not factor these into an interface instead? It should be more intuitive than a set of function pointers.